inference-engine/src/preprocessing/ie_preprocess_data.cpp

```diff
@@ -779,6 +779,7 @@ class PreProcessData : public IPreProcessData {
     Blob::Ptr getRoiBlob() const override;
 
     void execute(Blob::Ptr &outBlob, const PreProcessInfo& info, bool serial, int batchSize = -1) override;
```

Comment:
do we need a method with a single blob argument?

Reply:
not relevant any more - patch is reworked